src/commands/compile.ts

```diff
@@ -346,6 +351,10 @@ export async function importAndCompile(
             fileSystemProvider.fireFileChanged(file.uri);
           }
         }
+      } else if (file.uri.scheme === FILESYSTEM_SCHEMA || file.uri.scheme === FILESYSTEM_READONLY_SCHEMA) {
+        // Fire the file changed event to avoid VSCode alerting the user on the next folder-specific save (e.g. of settings.json) that
+        // "The content of the file is newer."
+        fileSystemProvider.fireFileChanged(file.unredirectedUri ?? file.uri);
```

Comment:
Is this specifically for the VS Code json files? I don't think Ive seen this problem otherwise.

Reply:
It showed up when using vscode.fs.copy() to put a testclass file into the server-side folder %UnitTest.Manager runs it from.